commons-vfs2/src/main/java/org/apache/commons/vfs2/provider/sftp/DummyPosixPermissions.java

```diff
@@ -0,0 +1,13 @@
+package org.apache.commons.vfs2.provider.sftp;
+
+import org.apache.commons.vfs2.util.PosixPermissions;
+
+/**
+ * Pretends that the current user is always the owner and in the same group.
+ */
+public class DummyPosixPermissions extends PosixPermissions {
```

Comment:
Thank you for your patch. Can you come up with a better class name here? Maybe one that reflects the fact that it "Pretends that the current user is always the owner and in the same group." "Dummy" does not reflect that IMO.

Reply:
LoggedInUserRWXPermissions? UserGroupImpersonatingPermissions? Open to suggestions here, I struggled and came up with dummy ;-)

Reply:
Simply _IsOwnerInGroupPosixPermissions_ ?